src/test/java/SeamsTest.java

```diff
@@ -0,0 +1,58 @@
+import ImageData.Energy;
+import ImageData.Seams;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SeamsTest {
+
+    @Test
+    void createHorizontalSeamArray() {
+        Energy testEnergy = Mockito.mock(Energy.class);
+        double[][] testEnergyArray = {
+                {1.0,4.0,3.0,5.0},
+                {3.0,2.0,1.0,6.0},
+                {2.0,5.0,4.0,1.0},
+                {4.0,3.0,2.0,1.0}};
+        Mockito.doReturn(testEnergyArray).when(testEnergy).getEnergyArray();
+        Seams testSeams = new Seams(testEnergy);
+
+        double[][] expectHorizontalSeams = {
+                {1.0,4.0,3.0,5.0},
+                {4.0,3.0,4.0,9.0},
+                {5.0,8.0,7.0,5.0},
+                {9.0,8.0,7.0,6.0}};
+
+
+        assertArrayEquals(expectHorizontalSeams,testSeams.getHorizontalSeamArray());
+    }
+
+    @Test
+    void createVerticalSeamArray() {
+        Energy testEnergy = Mockito.mock(Energy.class);
+        double[][] testEnergyArray = {
+                {1.0,4.0,3.0,5.0},
+                {3.0,2.0,1.0,6.0},
+                {2.0,5.0,4.0,1.0},
+                {4.0,3.0,2.0,1.0}};
+        Mockito.doReturn(testEnergyArray).when(testEnergy).getEnergyArray();
+        Seams testSeams = new Seams(testEnergy);
+
+        double[][] expectVerticalSeams = {
+                {1.0,3.0,2.0,4.0},
+                {5.0,3.0,7.0,5.0},
+                {6.0,4.0,7.0,7.0},
+                {9.0,10.0,5.0,8.0}};
```

Comment:
These numbers are wrong.

Reply:
I wrote the code so that it would flip the results - ie, the result is
{1, 5, 6, 9}
{3, 3, 4, 10}
{2, 7, 7, 5}
{4, 5, 7, 8}
I found it easier to write the calculateRemoveSeam method this way. Is that okay?